packages/firestore/src/core/listen_sequence.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ListenSequenceNumber } from './types';
+
+const LOG_TAG = 'ListenSequence';
+
+export interface SequenceNumberSyncer {
+  writeSequenceNumber(sequenceNumber: ListenSequenceNumber): void;
+  setSequenceNumberListener(
```

Comment:
We don't expose these setters in the other interfaces we have, leading me to think that we can likely drop it from this interface as well.

The name also suggests that there will be a receiving function.  I assume that's coming in a follow-up.

Reply:
Ah, ok, it looks like we expose writeable properties in the interface? I'll do that then.